guides/BlackboardLearnIntegrationGuide.md

```diff
@@ -107,14 +107,26 @@ _Note: openEQUELLA prepends a unique value (by **LTI Consumer**) to the user ID.
 7. Under **Tool Provider Information**, set the **Tool Provider URL** to `{https://your.oE.domain.xyz/demo/}ltisignon.do`.
 8. The **Tool Provider Key** and **Tool Provider Secret** will be preconfigured (and readonly).
 
+#### Alternative configuration
+New in oEQ 2020.2.0+, the LTI integration now supports leveraging the course folder structure in the selection session.
+
+To enable, use the steps above with the following changes.  Note this will require you to configure a REST integration flow (as described in this document) as well.
+
+1. For the URL, add the following parameters:  `{https://your.oE.domain.xyz/demo/}ltisignon.do??action=structured&connector_uuid={backing-oeq-external-system-connector}
```

Comment:
not for necassary this PR,
but sometime we should settle on a consistent {your domain goes here} link in the docs, that can be added to the allow list for link checking to make the valid/broken link checker happy.

Reply:
I added that to the agenda for the next CDM